src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -574,6 +574,31 @@ private Task SubscribeToGroup(string groupChannel, GroupData group)
             });
         }
 
+        public override Task InvokeConnectionsAsync(IReadOnlyList<string> connectionIds, string methodName, object[] args)
+        {
+            if (connectionIds == null)
+            {
+                throw new ArgumentNullException(nameof(connectionIds));
+            }
+            var publishTasks = new List<Task>(connectionIds.Count);
+            var message = new RedisInvocationMessage(target: methodName, arguments: args);
+
+            // If the connection is local we can skip sending the message through the bus since we require sticky connections.
+            // This also saves serializing and deserializing the message!
+            foreach(string connectionId in connectionIds)
+            {
+                var connection = _connections[connectionId];
+                if (connection != null)
+                {
+                    return connection.WriteAsync(message.CreateInvocation());
```

Comment:
This is going to exit from the whole method when you find a local connection, skipping all the remaining connections in the list. Instead, we should put the `PublishAsync` into an `else` clause and remove this `return`.

Reply:
Oops... Copy pasta error

Reply:
That's what I figured :)